src/conformal_cubed_sphere.jl

```diff
@@ -57,4 +57,48 @@ function conformal_cubed_sphere_mapping(x, y)
     return X, Y, Z
 end
 
+"""
+   lamrofnoc_cubed_sphere_mapping(X, Y, Z)
+
+Inverse mapping for conformal cube sphere for quadrant of north-pole face in which X and Y are both
+positive. All other mappings to other cube face coordinates can be recovered from rotations of this map.
```

Comment:
I guess it's not possible to specify an explicit range of valid inputs for `X, Y, Z`?

Would it be correct to say the range of inputs is the range of outputs from
```julia
X, Y, Z = conformal_cubed_sphere_mapping(x, y)
```
for `0 < x < 1` and `0 < y < 1`?